i6_models/primitives/feature_extraction.py

```diff
@@ -33,6 +45,11 @@ class LogMelFeatureExtractionV1Config(ModelConfiguration):
     num_filters: int
     center: bool
     n_fft: Optional[int] = None
+    periodic: bool = True
+    htk: bool = False
+    norm: Optional[Union[Literal["slaney"], float]] = "slaney"
+    dtype: DTypeLike = np.float32
+    spectrum_type: SpectrumType = SpectrumType.STFT
```

Comment:
> [curufinwe](https://github.com/curufinwe) approved these changes [8 minutes ago](https://github.com/rwth-i6/i6_models/pull/40#pullrequestreview-1769726515)
[curufinwe](https://github.com/curufinwe) merged commit [c363a01](https://github.com/rwth-i6/i6_models/commit/c363a0179aa6dfd34e48020e743c1ad3b86850b8) into main [8 minutes ago](https://github.com/rwth-i6/i6_models/pull/40#event-11179874625)

oh well 🤷

Reply:
I don't think the defaults are problematic here.

Reply:
It is not the defaults that are problematic, but adding new parameter automatically breaks setups. I will revert the PR. And then I would be in favor of Willis RasrCompatibleLogMelFeatureExtraction instead of the V2. But I would be fine with both.